com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -577,16 +591,18 @@ public void SetupLights(ScriptableRenderContext context, ref RenderingData rende
             RenderTargetHandle depthCopyTexture,
             RenderTargetHandle depthInfoTexture,
             RenderTargetHandle tileDepthInfoTexture,
-            RenderTargetHandle depthTexture,
-            RenderTargetHandle[] gbufferColorAttachments)
+            AttachmentDescriptor[] deferredInputs,
+            RenderTargetHandle depthTexture)
         {
             m_AdditionalLightsShadowCasterPass = additionalLightsShadowCasterPass;
             m_DepthCopyTexture = depthCopyTexture;
             m_DepthInfoTexture = depthInfoTexture;
             m_TileDepthInfoTexture = tileDepthInfoTexture;
-            m_GbufferColorAttachments = gbufferColorAttachments;
             m_DepthTexture = depthTexture;
-
+            deferredInputs[0] = GBufferDescriptors[0];
+            deferredInputs[1] = GBufferDescriptors[1];
+            deferredInputs[2] = GBufferDescriptors[2];
+            deferredInputs[3] = GBufferDescriptors[GBufferSliceCount];
```

Comment:
would be easier to understand if the indices would have names

Reply:
Yes, there are actually GBufferAlbedoIndex(), GBufferSpecularMetallicIndex(), GBufferNormalSmoothnessIndex(), ... that can be called instead.

Reply:
fixed in https://github.com/Unity-Technologies/Graphics/pull/396/commits/ae52519289b2320cf89f4b3244b22d9ea82df502